server/controllers/profile.js

```diff
@@ -121,4 +150,58 @@ exports.updateProfile = asyncHandler( async(req, res, next) => {
             res.status(200).send("Request updated successfully");
         })
     }
-});
\ No newline at end of file
+});
+
+exports.uploadImage =
```

Comment:
It looks like some of this code is part of a previous PR, you may have branched off a previous branch to work on this feature. Doing so is completely fine but in the future, please base your PRs properly so that only new changes are reflected.